validators/crypto_addresses/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""Crypto addresses."""
+# -*- coding: utf-8 -*-
+
+# isort: skip_file
+
+# local
+from .eth_address import eth_address
+from .trx_address import trx_address
```

Comment:
`crypto_addresses/trx_address.py` does not exist yet.

```diff
-from .trx_address import trx_address
```

Reply:
Done. [1715ea3](https://github.com/python-validators/validators/pull/276/commits/1715ea38098e3650c322aa73851d08a5044344b0)